cmd/lncli/commands.go

```diff
@@ -401,6 +401,26 @@ func walletBalance(ctx *cli.Context) error {
 	return nil
 }
 
+var ChannelBalanceCommand = cli.Command{
+	Name:        "channelbalance",
+	Description: "obtaining channel snapshots from all the peers and tallying the total available channel capacity within each channel",
```

Comment:
The description shouldn't leak the current implementation details. 

How about something more along the lines of: 
> Returns the sum of the total available channel balance across all open channels.